### PR TITLE
MRG: Allow complex TFR output

### PR DESCRIPTION
**Pull request by larsoner**
This allows for `EpochsTFR` to contain the (linear) complex outputs from `tfr_morlet`, like you can get from `tfr_array_morlet`. Eventually this could be useful for putting through an inverse, for example. In the meantime this incremental change is ready for review/merge from my end.

### CODE REVIEW

File: mne/time_frequency/tfr.py [hunk 1]
```diff
@@ -1534,6 +1541,24 @@ def __repr__(self):  # noqa: D105
         s += ', ~%s' % (sizeof_fmt(self._size),)
         return "<EpochsTFR  |  %s>" % s
 
+    def __abs__(self):
+        out = self.copy()
+        np.abs(out.data, out=out.data)
```
Comment (agramfort) [inline mne/time_frequency/tfr.py]: using out.data when you can from complex to float seems wrong.

do I miss something?

Comment (larsoner) [reply mne/time_frequency/tfr.py]: argh, no you're right

Comment (larsoner) [reply mne/time_frequency/tfr.py]: fixed

**Comment by agramfort** [comment]
merge when it's green

**Pull request merged**